src/main/java/com/google/api/codegen/py/PythonImportHandler.java

```diff
@@ -61,7 +66,7 @@ public PythonImportHandler(Interface service, ApiConfig apiConfig) {
     addImportExternal("google.gax", "path_template");
 
     // only if add enum import if there are enums
-    if (!Iterables.isEmpty(new PythonContextCommon().getEnumTypes(service.getModel()))) {
+    if (!Iterables.isEmpty(enumTable.getEnums())) {
```

Comment:
Acknowledged. You're right that PythonEnumSymbolTable doesn't make too much sense being owned by PythonImportHandler. I will make this change on Monday.